apolloschurchapp/index.js

```diff
@@ -1,12 +1,12 @@
-import './loadConfig';
-import { AppRegistry, YellowBox } from 'react-native';
-import SplashScreen from 'react-native-splash-screen';
-import ApollosConfig from '@apollosproject/config';
-import Storybook from './storybook';
+import "./loadConfig";
```

Comment:
Did eslint get messed up? Why are all of the single quotes `'` changing to double quotes `"`?

Reply:
My prettier setup is set to follow our eslint setup. Since eslint is apparently not setup prettier is doing whatever it wants 🤦‍♂ I've not set prettier to only make changes if eslint is setup but we'll still need to fix eslint in a different PR.